Sources/Testing/Support/Graph.swift

```diff
@@ -223,6 +223,55 @@ extension Graph {
     self.init(value: nil, children: children)
   }
 
+  /// Get or set the leaf value at the node identified by the specified sequence
+  /// of keys.
```

Comment:
I've seen this documented in [other](https://developer.apple.com/documentation/swift/dictionary/subscript(_:)-8rfql) mutable subscripts as "**Accesses** the value … for reading and writing", using the verb 'access'.

Reply:
The grammar gets a bit hard to parse if I change it because "for reading and writing" ends up quite far from "Accesses" so I'm just gonna leave it as-is.